app/default/data.py

```diff
@@ -121,6 +121,23 @@ def get_round_data(fund_id, round_id, as_dict=False):
         return round_response
 
 
+def get_round_data_by_short_names(fund_short_name, round_short_name):
+    request_url = Config.GET_ALL_ROUNDS_FOR_FUND_ENDPOINT.format(
+        fund_id=fund_short_name
+    )
+    response = get_data(
+        request_url, {"language": get_lang(), "use_short_name": "true"}
+    )
+    return next(
+        (
+            round
+            for round in response
+            if str.upper(round["short_name"]) == str.upper(round_short_name)
```

Comment:
There are some nuances with characters that can cause bugs if doing case comparison with `upper` or `lower` - so the standard way to do caseless matching is [casefold](https://docs.python.org/3/library/stdtypes.html#str.casefold).  I don't know if there will be any weird characters in the rounds `short_name` that will cause an issue, but it's good to use `casefold` as a standard anyways imo.

```suggestion
            if str.casefold(round["short_name"]) == str.casefold(round_short_name)
```